c++/src/kj/encoding.c++

```diff
@@ -410,7 +414,7 @@ String encodeUriComponent(ArrayPtr<const byte> bytes) {
     } else {
       result.add('%');
       result.add(HEX_DIGITS_URI[b/16]);
-      result.add(HEX_DIGITS_URI[b%16]);
```

Comment:
@kentonv I doubt it matters much when using modern compilers and modern hardware but I prefer `b & 15` over `b % 16` to get the bottom four bits of a uint8_t/kj::byte. I changed this in encodeHex and elsewhere in the file, but ready to revert it if you disagree.

Reply:
The compiler will certainly produce the same code either way. I prefer `%16` because it's easier for a human to think about, and it corresponds to the `/16` on the previous line (which you might argue should be `>>4` instead, but again, `/16` is easier for a human to read).